sdk/ml/azure-ai-ml/azure/ai/ml/entities/_component/_additional_includes.py

```diff
@@ -410,7 +410,19 @@ def merge_local_code_and_additional_includes(self) -> Path:
                 yield self.resolved_code_path.absolute()
             return
 
-        tmp_folder_path = Path(tempfile.mkdtemp())
+        # for now, upload path of a code asset will include the parent folder name of the code path
```

Comment:
why it will include the parent folder? 
Do we mean it will be something like this now?

code_with_additional_includes/code-a/flow.dag.yaml

Reply:
sorry, made a mistake in comment, only folder name will be included. I want to say parent name or folder name.

Reply:
it will be `code_with_additional_includes/flow.dag.yaml` now